app/main.go

```diff
@@ -96,9 +105,62 @@ func main() {
 		log.Fatalf("Invalid flags: %v", err)
 	}
 
-	http.HandleFunc("/", index)
-	http.HandleFunc("/sync", sync)
-	http.HandleFunc("/cleanup", cleanup)
+	config := tsbridge.NewConfig(&tsbridge.ConfigOptions{
+		Filename:                 *metricConfig,
+		MinPointAge:              *minPointAge,
+		CounterResetInterval:     *counterResetInterval,
+		SDLookBackInterval:       *sdLookBackInterval,
+		SDInternalMetricsProject: *sdInternalMetricsProject,
+		UpdateParallelism:        *updateParallelism,
+		EnableStatusPage:         *enableStatusPage,
+		StorageEngine:            *storageEngine,
+		SyncPeriod:               *syncPeriod,
+		SyncCleanupAfter:         *syncCleanupAfter,
+	})
+
+	http.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
+		index(w, r, config)
+	})
+	http.HandleFunc("/sync", func(w http.ResponseWriter, r *http.Request) {
+		syncHandler(w, r, config)
+	})
+	http.HandleFunc("/cleanup", func(w http.ResponseWriter, r *http.Request) {
+		cleanupHandler(w, r, config)
+	})
+
+	// Run a sync loop for standalone use
+	if env.NotAppEngine() {
```

Comment:
I think a separate function that does negation just makes the logic more confusing. `if !env.IsAppEngine()` is just as readable and short.

Reply:
This was added for another purpose which is no longer relevant, so agreed. Removing.